### ISSUE TITLE
linchpin-config.yml inconsistencies

### ISSUE THREAD

**Issue by greg-hellings** [CentOS-PaaS-SIG/linch-pin#202]
I have a linchpin_config.yml generated by linch-pin after asking me a series of questions. I have not edited it. Running `linchpin rise` gives me the following error
```
TASK [output_writer : Create output directories] *******************************
fatal: [localhost]: FAILED! => {"failed": true, "msg": "'inventoryfolder_path' is undefined"}
```
There is no definition of that file in the linchpin_config.yml file, though. Defining it in the linchpin_config.yml file causes execution to proceed as planned.

Note: this is on the `develop` branch.

**Comment by greg-hellings** [comment]
It might also be worth noting that there is a variable in linchpin_config.yml called `inventory_outputs_path`

**Comment by herlo** [comment]
@greg-hellings is this for the develop branch? I'm going to add some more labels so things can be tagged a bit better.

**Comment by herlo** [comment]
Closed by #206

**Comment by herlo** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Context Manager](https://trello.com/c/DMjTag2G/3-context-manager)
